static/js/index.js

```diff
@@ -0,0 +1,8 @@
+function myFunction() {
+  let copyText = document.getElementById("new_url")
+  navigator.clipboard.writeText(copyText.textContent);
+  alert("Copied the text: " + copyText.textContent);
```

Comment:
think I've seen something like this before somewhere - can't quite put my finger on it